src/task1/projection-module/projection_tests.py

```diff
@@ -0,0 +1,105 @@
+"""This is a few tests for the projection code. I includes both property and unit test.
+This is not intended to judge accurey just make sure nothing is
+really broken"""
+
+import pytest
+from hypothesis import given
+from hypothesis.strategies import floats
+from projection import ImageProjection
+import math as m
+import numpy as np
+
+
+def about(num1, num2, range) -> bool:
+    """Return true if num1 is range close to num2"""
+    return abs(num1 - num2) < range
```

Comment:
Can't you just assert all of the things calling about directly? Assert (abs(num1 - num2) < range)? idk this is what I'm used to for c# not sure if it's a different convention for py